cmd/juju/action/show.go

```diff
@@ -102,10 +102,21 @@ func (c *showCommand) Run(ctx *cmd.Context) error {
 			if !ok {
 				continue
 			}
-			args[argName] = actionArg{
-				Type:        infoMap["type"],
-				Description: infoMap["description"],
+
+			if infoMap["default"] != nil {
```

Comment:
You can check that the key exists and get the value if it exists in one step with something like:

			action := actionArg{
				Type:        infoMap["type"],
				Description: infoMap["description"],
			}
			if value, ok := infoMap["default"]; ok {
				action.Default = value
			}
			args[argName] = action

Reply:
Fixed per @jameinel comment in the next commit, thanks!